### ISSUE TITLE
Requiring `Enhance`

### ISSUE THREAD

**Issue by ppiekarczyk** [filamentgroup/politespace#8]
Please correct me if I'm wrong, but it looks like this is necessary for `politespace` to work:

```
<script>
	jQuery( function(){
		jQuery( document ).trigger( "enhance" );
	});
</script>
```
I pulled the demo and removed this and it all stopped working. It is not documented either. Can you tell me more about what it does?

Thank you

**Comment by jefflembeck** [comment]
There is an event listener bound to the document for the `enhance` event in order to initialize politespace. - https://github.com/filamentgroup/politespace/blob/master/src/politespace-init.js#L28

**Comment by zachleat** [comment]
We use this event to initialize both when the document first loads and when any new content is included with XMLHTTPRequest (such as with our AjaxInclude plugin). Many Filament Group plugins use this convention.

Please reopen if you have additional questions!

**Comment by josiahsprague** [comment]
It took me about an hour to figure out why politespace was not working for me. I was unfamiliar with the `enhance` event convention, and though I found this issue, I was trying to imitate the demo, and I had added a `data-enhanced` attribute to my input which prevented politespace from initializing. I think it would be useful to mention this in the usage section of the readme, maybe with a link to more info about the convention.
